test/integration/targets/nxos_vxlan_vtep/tests/common/sanity.yaml

```diff
@@ -23,6 +48,9 @@
         host_reachability: true
         source_interface: Loopback0
         source_interface_hold_down_time: 30
+        global_ingress_replication_bgp: "{{global_ingress_replication_bgp|default(omit)}}"
```

Comment:
Looks good overall. Just few syntax change needed in yaml. Please maintain space after `{{` and before `}}`.

Reply:
Done